core/actor/actor.hpp

```diff
@@ -0,0 +1,48 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef CPP_FILECOIN_CORE_ACTOR_ACTOR_HPP
+#define CPP_FILECOIN_CORE_ACTOR_ACTOR_HPP
+
+#include <libp2p/multi/content_identifier_codec.hpp>
+
+#include "primitives/big_int.hpp"
+
+namespace fc::actor {
+  using primitives::BigInt;
+  using libp2p::multi::ContentIdentifier;
+
+  struct Actor {
+    ContentIdentifier code;
+    ContentIdentifier head;
```

Comment:
Are there any benefits in making the Actor's public fields names differ from their respective counterparts in the spec https://filecoin-project.github.io/specs/#systems__filecoin_vm__actor ?

Reply:
Names taken from lotus https://github.com/filecoin-project/lotus/blob/master/chain/types/actor.go#L11